lib/flagsmith.rb

```diff
@@ -275,19 +286,28 @@ def process_identity_flags_from_api(identifier, traits = {})
       )
     end
 
-    def build_identity_model(identifier, traits = {})
+    # rubocop:disable Metrics/MethodLength
+    def get_identity_model(identifier, traits = {})
       unless environment
         raise Flagsmith::ClientError,
-              'Unable to build identity model when no local environment present.'
+              'Unable to get identity model when no local environment present.'
```

Comment:
🤔 I don't understand this restriction here. I get it's been here before, but other server-side SDKs actually don't have it.

_Maybe_ this check is relevant for local evaluation mode but then it should look for `config.local_evaluation? && environment`.

Reply:
I don't really know either. I think we should be confident that the `environment` is present but the case where it is not should error out anyway. That said, I don't see a big difference in keeping this as guard logic even if it wont ever be raised.